### ISSUE TITLE
The car is moving by itself in the empty world

### ISSUE THREAD

**Issue by brilianputraa** [chrissunny94/ackerman_ros_robot_gazebo_simulation#4]
Hi, I found that if I change the world of the gazebo to an empty world, the car is moving arbitrarily (it doesn't want to stay at the initial point), even if I don't give any control inputs. I have no idea what happens to the car, I try to look inside the urdf file but I cannot find the solution. You can see the video [here](https://youtu.be/d0nHrqw-Mpg)

I changed the car's initial position to:
```
  <arg name="x" default="0.0"/>
  <arg name="y" default="0.0"/>
  <arg name="z" default="0.035859"/>
  <arg name="roll" default="0.000401"/>
  <arg name="pitch" default="0.002969"/>
  <arg name="yaw" default="0.0"/>
```
Where the z value equals the default value

If I look at the odometry. Turns out the car has arbitrary linear velocity and angular velocity.
https://imgur.com/a/9f7zPxf

Is there any way that I can do to fix this issue? Thanks!

**Comment by chrissunny94** [comment]
Need to fix it

**Comment by brilianputraa** [comment]
Hi thanks for the reply. Is it the problem related with physics inside urdf?

**Comment by chrissunny94** [comment]
Yes i think the problem was because of Physics coefficients in the URDF model  .

I briefly remember the issue being 

1. Ground plane and wheels not interacting properly , so the **simulated-traction** was not proper .


I will probably port this code to current ROS2 .

I will leave this thread open for now .
